src/components/LandingPage.tsx

```diff
@@ -149,9 +155,9 @@ const LandingPage = props => {
   );
 };
 
-const mapStateToProps = state => {
+const mapStateToProps = (state: State) => {
```

Comment:
I think elsewhere we've started calling this function just `mapState`. We should settle on one naming convention.

```suggestion
const mapState = (state: State) => {
```